inc/alert.class.php

```diff
@@ -52,7 +52,9 @@ class Alert extends CommonDBTM {
    const ACTION      = 5;
    const PERIODICITY = 6;
 
-
+   /**
+    * @see CommonDBTM::prepareInputForAdd($input)
```

Comment:
@trasher I agree with you, 2) isn't the greatest solution. So 1 become the good solution :-)

You confirm (then I modify for the last time this PR ;-)) ?

Thanks for doc !